src/test/setup.js

```diff
@@ -9,13 +9,28 @@ import '@testing-library/jest-dom';
 // This installs jest matchers as a side effect as well.
 import fetchMock from 'fetch-mock-jest';
 import { Headers, Request, Response } from 'node-fetch';
+import { TextDecoder, TextEncoder } from 'util';
 
 jest.mock('../utils/worker-factory');
 import * as WorkerFactory from '../utils/worker-factory';
 import { autoMockResizeObserver } from './fixtures/mocks/resize-observer';
 
 autoMockResizeObserver();
 
+// Register TextDecoder and TextEncoder with the global scope.
+// These are now available globally in nodejs, but not when running with jsdom
+// in jest apparently.
+// Still let's double check that they're from the global scope as expected, so
+// that this can be removed in beforeEach below once it's implemented.
```

Comment:
Currently they are not in `beforeEach` block but this comment mentions that they are.

Also considering that we usually set global fields inside beforeEach, would it make sense to move it there? I don't have a preference though.

Reply:
Fun fact: I did this first, but there's this file https://github.com/firefox-devtools/profiler/blob/main/src/test/fixtures/example-symbol-table.js where `TextEncoder` is used right when the file is imported, so outside of a test. Then I realized that in this case where `TextDecoder` and `TextEncoder` are constructors and don't keep a test by themselves, we could just set them once outside of `beforeEach`.
Other things that are in `beforeEach` (not all though) are keeping  a state.

I'll update the comment!